src/sidebar.jsx

```diff
@@ -89,7 +105,24 @@ export const SidebarPanelDetails = ({ selected, path, setPath, setPathIndex, sho
                         <DescriptionListTerm>{_("Size")}</DescriptionListTerm>
                         <DescriptionListDescription>{cockpit.format("$0 $1", cockpit.format_bytes(selected.size), selected.size < 1000 ? "B" : "")}</DescriptionListDescription>
                     </DescriptionListGroup>}
+                    <DescriptionListGroup id="description-list-info">
```

Comment:
I would not put the info so down in the description list - I would rather see it as part of the title. Maybe with some smaller font.